metadata-ingestion-examples/mce-cli/README.md

```diff
@@ -0,0 +1,51 @@
+# MCE CLI
+
+A small application which can produce or consume MetadataChangeEvents.
```

Comment:
Can link MCE to this doc: https://github.com/linkedin/datahub/blob/master/docs/what/mxe.md